In [1]:
import pandas as pd
from datetime import datetime

#pd_fecha = pd.re]ad_html('https://apuestas.wplay.co/es/t/19311/Colombia-Primera-A')[0]
#pd_fecha

def equipo(lista):
    
    equipo=""
    for j in range(1,len(lista)-3):
        equipo+=lista[j]+" "
    return equipo[0:len(equipo)-1]

def pronostico(pd_fecha):
    FecList=[]
    GanaList=[]
    PronList=[]
    LocalList=[]
    VisitList=[]
    CuotaLocList=[]
    CuotaEmpList=[]
    CuotaVisList=[]

    #print (pd_fecha)
    #print (pd_fecha[3])
    
    for partido in range(len(pd_fecha)):
        
    
        local_list=pd_fecha[4][partido].split() #3
        #print(local_list)
        empate_list=pd_fecha[5][partido].split() #4
        #print (empate_list)
        visitante_list=pd_fecha[6][partido].split() #5
        #print (visitante_list)
        cuota_l=local_list[-2]
        cuota_e=empate_list[-2]
        cuota_v=visitante_list[-2]
        equipo_local=equipo(local_list)
        equipo_visitante=equipo(visitante_list)
    
        loc=float(cuota_l)
        emp=float(cuota_e)
        vis=float(cuota_v)
    
        pronostico=-0.805*loc-0.0517*emp+0.0125*vis+2.3923
    
        umbral=0.33
    
        if pronostico>umbral:
            gana="L"
        elif pronostico<0:
            gana="V"
        else:
            gana="E"
    
        print (pd_fecha[1][partido],gana,round(pronostico,2),equipo_local,"-",equipo_visitante,cuota_l,cuota_e,cuota_v,"     ")
        FecList.append( pd_fecha[1][partido])
        GanaList.append(gana)
        PronList.append(pronostico)
        LocalList.append(equipo_local)
        VisitList.append(equipo_visitante)
        CuotaLocList.append(loc)
        CuotaEmpList.append(emp)
        CuotaVisList.append(vis)
        
        
    resultados= {'hoy':datetime.now().strftime("%d/%m/%Y %H:%M"),
                 'Fecha':FecList,'LEV':GanaList,
        'Dif Pron':PronList,
        'Local':LocalList,
        'Visitante':VisitList,
        'C.Local':CuotaLocList,
        'C.Empate':CuotaEmpList,
        'C.Visitante':CuotaVisList
        }                       
    df=pd.DataFrame(resultados)
    
    return df

def fileout():

    from datetime import datetime
    now = datetime.now() # current date and time
    print (now)
    year = now.strftime("%Y")
    print("year:", year)

    month = now.strftime("%m")
    print("month:", month)

    day = now.strftime("%d")
    print("day:", day)

    time = now.strftime("%H:%M:%S")
    print("time:", time)

    #date_time = now.strftime("%m/%d/%Y, %H:%M:%S")
    date_time = now.strftime("%Y%m%d-%H%M") 
    salida=date_time+"-futcol.xls"
    print (salida)
    time_out=now.strftime("%d/%m/%Y %H:%M")
    print (time_out)
    #print("date and time:",date_time)
    return salida
    
#cars = {'Brand': ['Honda Civic','Toyota Corolla','Ford Focus','Audi A4'],'Price': [22000,25000,27000,35000]}
#df = pd.DataFrame(cars, columns = ['Brand', 'Price'])
     

In [2]:
link='https://apuestas.wplay.co/es/t/19311/Colombia-Primera-A' # Dejo de funcionar
# link='https://apuestas.wplay.co/es/futbol-colombia'

print ("COLOMBIA" ,"    ",link)
df=pronostico(pd.read_html(link)[0])
#print (pd.read_html(link)[0])
#print (df)
df.to_excel("output.xlsx")   

#now = datetime.now()
#date_time = now.strftime("%Y%m%d-%H%M") 
#date_time = datetime.now().strftime("%Y%m%d-%H%M") 

#salida= datetime.now().strftime("%Y%m%d-%H%M") +"-futcol.xls"

#print (salida)
df.to_excel(datetime.now().strftime("%Y%m%d-%H%M") +"-futcol.xls") 

# https://www.betclan.com/es/leagues/colombia-primera-a-predictions/200821/


COLOMBIA      https://apuestas.wplay.co/es/t/19311/Colombia-Primera-A
18:00 31 Mar L 0.58 Deportivo Pasto - Deportivo Pereira 2.10 3.25 3.95      
20:10 31 Mar L 0.9 Independiente Santa Fe - Boyaca Chico FC 1.727 3.45 5.75      
14:00 01 Abr E 0.29 Envigado FC - Once Caldas 2.45 3.20 3.20      
16:10 01 Abr E 0.3 Independiente Medellin - Tolima 2.45 3.10 3.20      
18:20 01 Abr V -0.03 La Equidad - Atlético Nacional 2.85 3.20 2.65      
20:30 01 Abr V -0.15 Atletico Bucaramanga - Millonarios 3.00 3.10 2.60      
16:00 02 Abr L 1.0 América de Cali - Jaguares De Córdoba 1.571 4.00 6.50      
18:10 02 Abr V -0.31 Union Magdalena - Rionegro Aguilas 3.20 3.10 2.45      
20:20 02 Abr V -0.08 Atletico Huila - Deportivo Cali 2.90 3.25 2.55      
20:10 03 Abr L 1.03 CD Junior - Alianza Petrolera 1.55 4.00 7.00      


In [9]:
link='https://www.futbolred.com/tabla-de-posiciones/liga-betplay'
#link='https://apuestas.wplay.co/es/futbol-colombia'

print ("POSOICIONES" ,"    ",link)
#lst=pd.read_html(link)#[1]

#df = pd.DataFrame(lst)

#print (df)
#df.to_excel("output_positions.xlsx") 
#df.to_excel(datetime.now().strftime("%Y%m%d-%H%M") +"-CopAmerica.xls")


POSOICIONES      https://www.futbolred.com/tabla-de-posiciones/liga-betplay


In [5]:
loc=  1.68
emp=  3.75
vis=   5
DifGol=-0.805*loc-0.0517*emp+0.0125*vis+2.3923
print (DifGol)



0.908525


In [1]:

2.1+1.75+1.01 +0.47 - 5
178/(178+276)

900*800000


720000000

#### GOLPREDICTOR

### FECHA 9 (1/3)

20:00 23 Feb L 0.75 CD Junior - Millonarios 1.90 3.20 4.50      
14:00 24 Feb L 1.22 Independiente Santa Fe - Boyaca Chico FC 1.31 4.85 11.00      
16:00 24 Feb L 0.45 Rionegro Aguilas - La Equidad 2.30 2.75 3.75      
18:05 24 Feb V -0.57 Atletico Bucaramanga - Atletico Nacional 3.50 3.30 2.10      
20:10 24 Feb L 0.97 America de Cali - Envigado FC 1.63 3.55 6.00      
14:00 25 Feb E 0.15 Once Caldas - Deportivo Pasto 2.65 2.875 3.00      
16:00 25 Feb L 0.51 Jaguares De Cordoba - Deportivo Pereira 2.20 3.00 3.65      
18:05 25 Feb V -0.84 Alianza Petrolera - Tolima 3.85 3.10 2.10      
20:10 25 Feb V -0.13 Independiente Medellin - Deportivo Cali 3.00 2.80 2.70      
17:30 27 Feb L 0.91 La Equidad - Boyaca Chico FC 1.727 3.20 6.00      
20:00 27 Feb V -0.05 Patriotas Boyaca - CD Junior 2.875 3.10 2.60      
14:00 28 Feb V -0.44 Envigado FC - Independiente Medellin 3.35 3.10 2.30      
16:00 28 Feb L 0.9 Tolima - Rionegro Aguilas 1.75 3.10 6.00      
18:05 28 Feb L 0.35 Deportivo Pasto - America de Cali 2.375 3.20 3.10      
20:10 28 Feb L 0.53 Deportivo Cali - Independiente Santa Fe 2.20 2.80 4.00      

20:00 23 Feb L 0.55 CD Junior - Millonarios 2.15 3.10 3.70      
14:00 24 Feb L 1.2 Independiente Santa Fe - Boyaca Chico FC 1.33 4.75 10.00      
16:00 24 Feb L 0.45 Rionegro Aguilas - La Equidad 2.30 2.75 3.75      
18:05 24 Feb V -0.57 Atletico Bucaramanga - Atletico Nacional 3.50 3.30 2.10      
20:10 24 Feb L 1.02 America de Cali - Envigado FC 1.56 3.80 6.50      
14:00 25 Feb E 0.15 Once Caldas - Deportivo Pasto 2.65 2.875 3.00      
16:00 25 Feb L 0.39 Jaguares De Cordoba - Deportivo Pereira 2.35 2.95 3.40      
18:05 25 Feb V -0.84 Alianza Petrolera - Tolima 3.85 3.10 2.10      
20:10 25 Feb V -0.47 Independiente Medellin - Deportivo Cali 3.40 2.95 2.35      

### FECHA 8 (1/3)

14:00 21 Feb L 0.43 La Equidad - Once Caldas 2.30 2.95 3.55      
16:00 21 Feb L 1.02 Deportivo Pasto - Jaguares De Cordoba 1.58 3.60 6.50      
18:05 21 Feb L 1.05 Independiente Santa Fe - Independiente Medellin 1.53 3.80 7.00      
20:10 21 Feb L 0.59 Atletico Nacional - America de Cali 2.10 3.10 3.85   

14:00 21 Feb E 0.32 La Equidad - Once Caldas 2.45 2.80 3.40      
16:00 21 Feb L 1.02 Deportivo Pasto - Jaguares De Cordoba 1.58 3.60 6.50      
18:05 21 Feb L 0.91 Independiente Santa Fe - Independiente Medellin 1.73 3.15 6.25      
20:10 21 Feb L 0.67 Atletico Nacional - America de Cali 2.00 3.10 4.20  


17:30 20 Feb V -0.4 Deportivo Pereira - Rionegro Aguilas 3.30 3.10 2.30      
20:00 20 Feb V -0.78 Boyaca Chico FC - CD Junior 3.80 2.75 2.30      

15:15 20 Feb E 0.19 Envigado FC - Atletico Bucaramanga 2.60 2.80 3.10      
17:30 20 Feb V -0.06 Deportivo Pereira - Rionegro Aguilas 2.90 3.00 2.625      
20:00 20 Feb V -0.78 Boyaca Chico FC - CD Junior 3.80 2.75 2.30      
14:00 21 Feb L 0.36 La Equidad - Once Caldas 2.40 2.85 3.45      
16:00 21 Feb L 1.02 Deportivo Pasto - Jaguares De Cordoba 1.58 3.60 6.50      
18:05 21 Feb L 0.91 Independiente Santa Fe - Independiente Medellin 1.73 3.15 6.25      
20:10 21 Feb L 0.67 Atletico Nacional - America de Cali 2.00 3.10 4.20      
20:00 22 Feb L 1.0 Deportivo Cali - Alianza Petrolera 1.61 3.50 6.50      

20:00 19 Feb L 1.14 Tolima - Patriotas Boyaca 1.43 4.10 8.50      
15:15 20 Feb E 0.27 Envigado FC - Atletico Bucaramanga 2.50 2.875 3.20      
17:30 20 Feb E 0.14 Deportivo Pereira - Rionegro Aguilas 2.65 2.95 2.90      
20:00 20 Feb V -1.76 Boyaca Chico FC - CD Junior 5.00 2.85 1.96      
14:00 21 Feb L 0.48 La Equidad - Once Caldas 2.25 2.80 3.80      
16:00 21 Feb L 1.02 Deportivo Pasto - Jaguares De Cordoba 1.58 3.60 6.50      
18:05 21 Feb L 0.9 Independiente Santa Fe - Independiente Medellin 1.76 3.00 6.25      
20:10 21 Feb L 0.52 Atletico Nacional - America de Cali 2.21 2.80 4.00      
20:00 22 Feb L 0.97 Deportivo Cali - Alianza Petrolera 1.63 3.60 6.00  


### FECHA 7 (1/3)

20:05 18 Feb V -0.01 America de Cali - Independiente Santa Fe 2.85 2.75 2.875

16:00 18 Feb E 0.18 Jaguares De Cordoba - La Equidad 2.625 2.75 3.20      
18:00 18 Feb L 0.78 Millonarios - Deportivo Pasto 1.86 3.30 4.50      
20:05 18 Feb E 0.23 America de Cali - Independiente Santa Fe 2.55 2.80 3.20   

16:00 18 Feb E 0.04 Jaguares De Cordoba - La Equidad 2.80 2.70 3.00      
18:00 18 Feb L 0.73 Millonarios - Deportivo Pasto 1.93 3.20 4.33      
20:05 18 Feb E 0.23 America de Cali - Independiente Santa Fe 2.55 2.85 3.20      
    
16:00 18 Feb V -0.01 Jaguares De Cordoba - La Equidad 2.85 2.80 2.875      
18:00 18 Feb L 0.75 Millonarios - Deportivo Pasto 1.91 3.20 4.50      
20:05 18 Feb E 0.28 America de Cali - Independiente Santa Fe 2.50 2.80 3.25      

14:00 17 Feb L 0.56 Atletico Bucaramanga - Deportivo Pereira 2.13 3.20 3.60      
16:00 17 Feb L 0.83 Rionegro Aguilas - Envigado FC 1.81 3.25 5.00      
18:05 17 Feb L 0.97 Independiente Medellin - Boyaca Chico FC 1.63 3.60 6.00      
20:10 17 Feb L 0.47 CD Junior - Deportivo Cali 2.25 3.00 3.55   

14:00 17 Feb L 0.56 Atletico Bucaramanga - Deportivo Pereira 2.13 3.20 3.60      
16:00 17 Feb L 0.85 Rionegro Aguilas - Envigado FC 1.78 3.35 5.00      
18:05 17 Feb L 1.01 Independiente Medellin - Boyaca Chico FC 1.59 3.60 6.50      
20:10 17 Feb L 0.55 CD Junior - Deportivo Cali 2.15 3.00 3.85      


20:00 19 Feb L 1.15 Tolima - Patriotas Boyaca 1.40 4.333 9.00      
15:15 20 Feb E 0.28 Envigado FC - Atletico Bucaramanga 2.50 2.75 3.35      
17:30 20 Feb E 0.19 Deportivo Pereira - Rionegro Aguilas 2.60 2.80 3.10      
20:00 20 Feb V -1.95 Boyaca Chico FC - CD Junior 5.25 2.80 1.95      
14:00 21 Feb L 0.53 La Equidad - Once Caldas 2.20 2.80 4.00      
16:00 21 Feb L 1.08 Deportivo Pasto - Jaguares De Cordoba 1.50 3.80 7.50      
18:05 21 Feb L 0.97 Independiente Santa Fe - Independiente Medellin 1.65 3.40 6.50      
20:10 21 Feb L 0.61 Atletico Nacional - America de Cali 2.10 2.875 4.20      
20:00 22 Feb L 1.05 Deportivo Cali - Alianza Petrolera 1.533 3.85 7.00      


### FECHA 6 (1/3)

20:10 14 Feb E 0.12 Deportivo Pasto - Tolima 2.70 2.625 3.25      

15:15 13 Feb V -0.96 La Equidad - Millonarios 4.00 3.10 2.03      
17:30 13 Feb L 0.84 Deportivo Cali - Rionegro Aguilas 1.81 3.10 5.25      
20:00 13 Feb V -1.37 Envigado FC - CD Junior 4.50 3.10 1.93      

Voy con estos 3
16:00 12 Feb L 0.6 Deportivo Pereira - Patriotas Boyaca 2.11 2.90 4.10      
18:00 12 Feb L 0.42 Alianza Petrolera - Once Caldas 2.30 3.10 3.30      
20:05 12 Feb V -1.76 Boyaca Chico FC - America de Cali 5.00 2.85 1.96   

16:00 12 Feb L 0.62 Deportivo Pereira - Patriotas Boyaca 2.08 2.90 4.20      
18:00 12 Feb L 0.46 Alianza Petrolera - Once Caldas 2.25 3.10 3.45      
20:05 12 Feb V -1.54 Boyaca Chico FC - America de Cali 4.75 2.65 2.11      
15:15 13 Feb V -0.96 La Equidad - Millonarios 4.00 3.10 2.03      
17:30 13 Feb L 0.84 Deportivo Cali - Rionegro Aguilas 1.81 3.10 5.50      
20:00 13 Feb V -1.37 Envigado FC - CD Junior 4.50 3.10 1.93      
14:00 14 Feb L 1.08 Independiente Santa Fe - Atletico Bucaramanga 1.49 3.90 7.50      
18:05 14 Feb L 1.02 Independiente Medellin - Jaguares De Cordoba 1.59 3.50 6.75      
20:10 14 Feb V -0.08 Deportivo Pasto - Tolima 2.95 2.65 2.90   

16:00 12 Feb L 0.56 Deportivo Pereira - Patriotas Boyaca 2.16 2.80 4.15      
18:00 12 Feb E 0.27 Alianza Petrolera - Once Caldas 2.50 2.95 3.10      
20:05 12 Feb V -1.95 Boyaca Chico FC - America de Cali 5.25 2.80 1.96      
15:15 13 Feb V -0.96 La Equidad - Millonarios 4.00 3.10 2.05      
17:30 13 Feb L 0.84 Deportivo Cali - Rionegro Aguilas 1.81 3.10 5.50      
20:00 13 Feb V -1.37 Envigado FC - CD Junior 4.50 3.10 1.93      
14:00 14 Feb L 1.08 Independiente Santa Fe - Atletico Bucaramanga 1.49 3.90 7.50      
18:05 14 Feb L 0.97 Independiente Medellin - Jaguares De Cordoba 1.66 3.30 6.50      
20:10 14 Feb V -0.08 Deportivo Pasto - Tolima 2.95 2.65 2.90      

16:00 12 Feb L 0.56 Deportivo Pereira - Patriotas Boyaca 2.16 2.80 4.15      
18:00 12 Feb E 0.19 Alianza Petrolera - Once Caldas 2.60 2.85 3.10      
20:05 12 Feb V -1.95 Boyaca Chico FC - America de Cali 5.25 2.80 1.96      
15:15 13 Feb V -0.63 La Equidad - Millonarios 3.60 2.90 2.30      
17:30 13 Feb L 0.86 Deportivo Cali - Rionegro Aguilas 1.79 3.10 5.50      
20:00 13 Feb V -1.37 Envigado FC - CD Junior 4.50 3.10 1.93      
14:00 14 Feb L 1.01 Independiente Santa Fe - Atletico Bucaramanga 1.59 3.60 6.50      
18:05 14 Feb L 0.99 Independiente Medellin - Jaguares De Cordoba 1.63 3.40 6.50      
20:10 14 Feb V -0.0 Deportivo Pasto - Tolima 2.85 2.625 3.00     

### FECHA 5 (9/9)


19:40 09 Feb E 0.23 Rionegro Aguilas - Deportivo Pasto 2.55 2.875 3.10   
19:40 09 Feb E 0.19 Rionegro Aguilas - Deportivo Pasto 2.60 2.90 3.00      
19:40 09 Feb L 0.35 Rionegro Aguilas - Deportivo Pasto 2.40 2.90 3.30    

20:00 08 Feb E 0.03 Atletico Bucaramanga - La Equidad 2.80 2.75 3.00   
20:00 08 Feb E 0.07 Atletico Bucaramanga - La Equidad 2.75 2.85 2.90 
20:00 08 Feb E 0.12 Atletico Bucaramanga - La Equidad 2.70 2.70 3.10   

19:40 09 Feb L 0.35 Rionegro Aguilas - Deportivo Pasto 2.40 2.90 3.30      
16:00 12 Feb L 0.65 Deportivo Pereira - Patriotas Boyaca 2.05 2.85 4.50      
18:00 12 Feb E 0.17 Alianza Petrolera - Once Caldas 2.625 2.80 3.10      
20:05 12 Feb V -1.35 Boyaca Chico FC - America de Cali 4.50 2.85 2.05      
15:15 13 Feb V -0.7 La Equidad - Millonarios 3.70 2.80 2.30      
17:30 13 Feb L 0.82 Deportivo Cali - Rionegro Aguilas 1.85 3.00 5.50      
20:00 13 Feb V -1.36 Envigado FC - CD Junior 4.50 3.00 2.00      
14:00 14 Feb L 0.98 Independiente Santa Fe - Atletico Bucaramanga 1.62 3.60 6.00      
18:05 14 Feb L 0.69 Independiente Medellin - Jaguares De Cordoba 2.00 2.95 4.50      
20:10 14 Feb V -0.04 Deportivo Pasto - Tolima 2.90 2.55 3.10    

OK-CON ESTE
14:00 07 Feb L 1.13 Millonarios - Deportivo Pereira 1.40 4.50 8.00      
16:00 07 Feb L 1.01 Tolima - Independiente Medellin 1.55 4.00 6.00      
18:05 07 Feb V -0.56 Patriotas Boyaca - Deportivo Cali 3.50 3.10 2.25      
20:10 07 Feb V -0.75 Jaguares De Cordoba - Independiente Santa Fe 3.75 2.95 2.20  

20:00 08 Feb E 0.12 Atletico Bucaramanga - La Equidad 2.70 2.70 3.10      
19:40 09 Feb L 0.35 Rionegro Aguilas - Deportivo Pasto 2.40 2.90 3.30      
16:00 12 Feb L 0.65 Deportivo Pereira - Patriotas Boyaca 2.05 2.85 4.50      
18:00 12 Feb E 0.17 Alianza Petrolera - Once Caldas 2.625 2.80 3.10      
20:05 12 Feb V -1.58 Boyaca Chico FC - America de Cali 4.75 3.25 1.85      
15:15 13 Feb V -0.58 La Equidad - Millonarios 3.55 2.80 2.375      
17:30 13 Feb L 0.96 Deportivo Cali - Rionegro Aguilas 1.67 3.25 6.50      
20:00 13 Feb V -3.18 Envigado FC - CD Junior 6.75 3.10 1.70      
14:00 14 Feb L 1.06 Independiente Santa Fe - Atletico Bucaramanga 1.533 3.70 7.50      
18:05 14 Feb L 0.81 Independiente Medellin - Jaguares De Cordoba 1.85 3.00 5.25      
20:10 14 Feb V -0.04 Deportivo Pasto - Tolima 2.90 2.55 3.10      


15:15 06 Feb L 0.75 Once Caldas - Envigado FC 1.91 3.20 4.50      
17:30 06 Feb L 1.15 CD Junior - Alianza Petrolera 1.43 4.00 9.00      
20:00 06 Feb L 1.31 Atletico Nacional - Boyaca Chico FC 1.23 5.50 15.00      
14:00 07 Feb L 1.08 Millonarios - Deportivo Pereira 1.47 4.20 7.00      
16:00 07 Feb L 0.63 Tolima - Independiente Medellin 2.05 3.10 4.00      
18:05 07 Feb V -0.56 Patriotas Boyaca - Deportivo Cali 3.50 3.20 2.15      
20:10 07 Feb V -0.58 Jaguares De Cordoba - Independiente Santa Fe 3.55 2.80 2.375      
20:00 08 Feb E 0.12 Atletico Bucaramanga - La Equidad 2.70 2.70 3.10      
19:40 09 Feb L 0.35 Rionegro Aguilas - Deportivo Pasto 2.40 2.875 3.40      
16:00 12 Feb L 0.53 Deportivo Pereira - Patriotas Boyaca 2.20 2.80 4.20      
18:00 12 Feb E 0.28 Alianza Petrolera - Once Caldas 2.50 2.75 3.40     

### FECHA 4 (4/9) +0.33

20:00 04 Feb V -0.06 Independiente Medellin - Millonarios 2.90 2.90 2.70 

15:15 06 Feb L 0.75 Once Caldas - Envigado FC 1.91 3.10 4.75      
17:30 06 Feb L 1.08 CD Junior - Alianza Petrolera 1.51 3.75 7.50      
20:00 06 Feb L 1.25 Atletico Nacional - Boyaca Chico FC 1.286 5.00 12.00    

14:00 04 Feb L 1.1 Independiente Santa Fe - Patriotas Boyaca 1.47 4.00 7.50      
20:00 04 Feb V -0.1 Independiente Medellin - Millonarios 2.95 2.90 2.65  

14:00 03 Feb V -0.03 La Equidad - CD Junior 2.875 2.85 2.80      
16:00 03 Feb V -0.97 Deportivo Pereira - Tolima 4.00 3.30 1.97      
18:05 03 Feb L 0.39 Alianza Petrolera - Rionegro Aguilas 2.35 3.00 3.35      
20:10 03 Feb L 1.07 America de Cali - Atletico Bucaramanga 1.50 4.00 7.00      


14:00 03 Feb E 0.07 La Equidad - CD Junior 2.75 2.85 2.90      
16:00 03 Feb V -0.8 Deportivo Pereira - Tolima 3.80 3.15 2.10      
18:05 03 Feb L 0.39 Alianza Petrolera - Rionegro Aguilas 2.35 3.00 3.35      
20:10 03 Feb L 1.07 America de Cali - Atletico Bucaramanga 1.50 4.00 7.00   

14:00 03 Feb V -0.09 La Equidad - CD Junior 2.95 2.80 2.75      
16:00 03 Feb V -0.61 Deportivo Pereira - Tolima 3.55 3.25 2.15      
18:05 03 Feb E 0.22 Alianza Petrolera - Rionegro Aguilas 2.55 3.00 2.95      
20:10 03 Feb L 0.91 America de Cali - Atletico Bucaramanga 1.71 3.40 5.50    


20:05 02 Feb E 0.3 Deportivo Pasto - Atletico Nacional 2.45 3.10 3.00      

16:00 02 Feb E 0.24 Envigado FC - Jaguares De Cordoba 2.55 2.75 3.20      
18:00 02 Feb V -1.22 Boyaca Chico FC - Deportivo Cali 4.33 3.00 2.01      
20:05 02 Feb L 0.38 Deportivo Pasto - Atletico Nacional 2.35 3.20 3.20  

16:00 02 Feb E 0.18 Envigado FC - Jaguares De Cordoba 2.625 2.75 3.20      
18:00 02 Feb V -1.22 Boyaca Chico FC - Deportivo Cali 4.33 3.00 2.01      
20:05 02 Feb E 0.3 Deportivo Pasto - Atletico Nacional 2.45 3.10 3.00  

16:00 02 Feb E 0.24 Envigado FC - Jaguares De Cordoba 2.55 2.80 3.25      
18:00 02 Feb V -1.36 Boyaca Chico FC - Deportivo Cali 4.50 3.00 2.01      
20:05 02 Feb E 0.18 Deportivo Pasto - Atletico Nacional 2.60 3.00 2.95      



FEBRERO 1
16:00 02 Feb E 0.18 Envigado FC - Jaguares De Cordoba 2.625 2.70 3.25      
18:00 02 Feb V -1.15 Boyaca Chico FC - Deportivo Cali 4.25 2.85 2.11      
20:05 02 Feb V -0.22 Deportivo Pasto - Atletico Nacional 3.10 2.95 2.55

14:00 03 Feb V -0.29 La Equidad - CD Junior 3.20 2.70 2.65      
16:00 03 Feb V -0.69 Deportivo Pereira - Tolima 3.65 3.25 2.09      
16:00 03 Feb L 1.1 Independiente Santa Fe - Patriotas Boyaca 1.45 4.25 7.50      
18:05 03 Feb E 0.22 Alianza Petrolera - Rionegro Aguilas 2.55 3.00 2.95      
20:10 03 Feb L 0.9 America de Cali - Atletico Bucaramanga 1.72 3.35 5.50      
20:00 04 Feb E 0.03 Independiente Medellin - Millonarios 2.80 2.75 2.90    

### FECHA 3 (6/9) +0.32

14:00 01 Feb L 0.93 Millonarios - Once Caldas 1.68 3.50 5.60 
 
19:40 31 Ene ExV 0.3 Atletico Bucaramanga - Independiente Medellin 2.45 3.00 3.20      
19:40 31 Ene ExV 0.16 Atletico Bucaramanga - Independiente Medellin 2.625 2.95 2.95 

15:30 31 Ene L 0.91 Tolima - La Equidad 1.71 3.40 5.50     
16:00 30 Ene V -0.22 Rionegro Aguilas - Independiente Santa Fe 3.10 2.875 2.55      
18:05 30 Ene L 1.02 Atletico Nacional - Deportivo Pereira 1.57 3.70 6.50      
20:10 30 Ene L 0.83 CD Junior - America de Cali 1.81 3.25 5.00 

14:00 30 Ene LxE 0.51 Patriotas Boyaca - Boyaca Chico FC 2.20 3.00 3.75 
20:00 29 Ene LxE 1.12 Deportivo Cali - Envigado FC 1.43 4.25 8.00      
18:00 29 Ene ExL 0.18 Jaguares De Cordoba - Alianza Petrolera 2.60 2.95 2.95      

ENERO 26 - umbral=0.33

18:00 29 Ene E 0.31 Jaguares De Cordoba - Alianza Petrolera 2.45 2.95 3.25      
20:00 29 Ene L 1.18 Deportivo Cali - Envigado FC 1.41 4.00 10.00      
14:00 30 Ene L 0.35 Patriotas Boyaca - Boyaca Chico FC 2.40 3.00 3.25      
16:00 30 Ene V -0.33 Rionegro Aguilas - Independiente Santa Fe 3.25 2.625 2.70      
18:05 30 Ene L 0.99 Atletico Nacional - Deportivo Pereira 1.59 3.75 6.00      
20:10 30 Ene L 0.59 CD Junior - America de Cali 2.13 2.80 4.25      
15:30 31 Ene L 0.8 Tolima - La Equidad 1.86 3.10 5.00      
19:40 31 Ene E 0.22 Atletico Bucaramanga - Independiente Medellin 2.55 3.00 2.95      
18:00 01 Feb L 0.75 Millonarios - Once Caldas 1.91 3.20 4.50      

### FECHA 2 (4/7) +1.05

14:00 25 Ene VxL -0.31 La Equidad - Atletico Nacional 3.20 3.10 2.35 
14:00 25 Ene VxL -0.69 La Equidad - Atletico Nacional 3.65 3.20 2.10      


17:30 25 Ene V -0.85 Once Caldas - CD Junior 3.85 3.20 2.05      
17:30 25 Ene V -0.55 Once Caldas - CD Junior 3.50 3.00 2.25      

16:00 24 Ene LxE 0.72 America de Cali - Rionegro Aguilas 1.94 3.20 4.33      

18:05 24 Ene ExL 0.35 Jaguares De Cordoba - Atletico Bucaramanga 2.40 3.00 3.25   Casi Local?..
18:05 24 Ene ExL 0.15 Jaguares De Cordoba - Atletico Bucaramanga 2.65 2.875 2.95  Me quedo con este

20:10 24 Ene V -0.89 Deportivo Pereira - Deportivo Cali 3.90 3.20 2.05   

17:35 23 Ene E 0.02 Envigado FC - Deportivo Pasto 2.80 3.00 2.70      ??????
17:35 23 Ene V -0.15 Envigado FC - Deportivo Pasto 3.00 3.10 2.45     Me quedo con este.. y OK

20:00 22 Ene L 1.08 Independiente Medellin - Patriotas Boyaca 1.50 3.90 7.50 


### FECHA 1 (5/8) +0.59
19:40 20 Ene LxE 0.55 Alianza Petrolera - Deportivo Pereira 2.15 3.10 3.65  
19:40 19 Ene L 0.85 Atletico Bucaramanga - Boyaca Chico FC 1.78 3.30 5.00      
15:30 18 Ene LxE 0.73 Tolima - Once Caldas 1.93 3.20 4.33      
14:00 17 Ene LxE 0.35 Deportivo Pasto - La Equidad 2.40 2.875 3.35      
18:05 17 Ene L 0.48 Atletico Nacional - Independiente Santa Fe 2.25 2.90 3.70      
20:10 17 Ene L 1.11 Deportivo Cali - Jaguares De Cordoba 1.46 4.00 8.00      
17:00 16 Ene L 1.1 Millonarios - Envigado FC 1.47 4.00 7.50      
19:30 16 Ene L 0.96 CD Junior - Independiente Medellin 1.63 3.65 5.75      


## 2020-11-22 -B

10 PARTIDOS WPLAY (3000 CU)   CODERE (3000)

### COLOMBIA (-2 )
18:05 22 Nov L 0.54 CD Junior - Tolima 2.15 3.20 3.60   
20:10 22 Nov E 0.05 Deportivo Pasto - Independiente Santa Fe 2.75 3.10 2.65   

### ESPAÑA (-3 )
12:30 22 Nov (1-3 V) L 0.51 Granada - Real Valladolid 2.20 3.00 3.60   
15:00 22 Nov E 0.22 CD Alaves - Valencia 2.55 3.10 2.90   
15:00 23 Nov L 0.58 Athletic Bilbao - Real Betis 2.10 3.25 3.60   

### ITALIA (-2 + 1.80 )
12:00 22 Nov (1-0 L) L 0.82 Udinese - Génova 1.80 3.50 4.50   
14:45 22 Nov L 0.49 Napoles - AC Milán 2.20 3.40 3.25   

### INGLATERRA (-3 )
14:15 22 Nov L 0.71 Liverpool - Leicester City 1.90 3.85 3.70   
12:30 23 Nov E 0.14 Burnley - Crystal Palace 2.65 3.00 2.85   
15:00 23 Nov L 0.42 Wolverhampton Wanderers - Southampton 2.30 3.20 3.25   

## 2020-11-22 -A (+0.79)

10 PARTIDOS WPLAY (34576 3000 CU)  CODERE (41619 3000 CU varios 1000)   

### ESPAÑA (-0.25)
08:00 22 Nov (0-0 E) V -0.42 SD Eibar - Getafe 3.35 2.80 2.45   (8000 codere)   
10:15 22 Nov (0-1 V) V -1.4 CF Cadiz - Real Sociedad 4.50 3.70 1.75  (8000 codere)   

### ITALIA (-0.07)
06:30 22 Nov (0-1 V) L 0.99 Fiorentina - Benevento Calcio 1.55 4.33 5.50    
09:00 22 Nov (0-2 V) V -0.27 Verona - Sassuolo 3.10 3.70 2.10   
09:00 22 Nov (1-2 V) E 0.24 Sampdoria - Bologna 2.50 3.45 2.70   
09:00 22 Nov (4-2 L) L 1.13 Inter - Torino 1.333 5.50 8.00   
09:00 22 Nov (3-0 L) L 1.03 Roma - Parma 1.50 4.50 6.00      

### INGLATERRA (+1.109)
07:00 22 Nov (2-3 V) V -0.83 Fulham - Everton 3.80 3.70 1.909   
09:00 22 Nov (0-1 V) V -0.4 Sheffield United - West Ham United 3.30 3.25 2.20   
11:30 22 Nov (0-0 E) V -0.26 Leeds United - Arsenal 3.10 3.60 2.20   

## 2020-11-21 

13 PARTIDOS WPLAY (31716 2000 CU)  CODERE (38519 3000 CU ULTIMO 2000) 

### COLOMBIA
17:30 21 Nov E 0.17 La Equidad - Deportivo Cali 2.60 3.10 2.80   
20:00 21 Nov E 0.21 America de Cali - Atletico Nacional 2.55 3.25 2.75   

### ESPAÑA
08:00 21 Nov L(1-1) 0.91 Levante - Elche 1.67 3.90 4.95   
10:15 21 Nov E 0.06 Villarreal - Real Madrid 2.70 3.65 2.40   
12:30 21 Nov L 0.87 Sevilla - Celta de Vigo 1.75 3.50 5.00   
15:00 21 Nov V -0.01 Atlético Madrid - Barcelona 2.80 3.40 2.45   

### ITALIA
09:00 21 Nov V -1.41 Crotone - Lazio 4.50 3.90 1.73   
12:00 21 Nov V -2.87 Spezia - Atalanta 6.25 4.75 1.45   
14:45 21 Nov L 1.14 Juventus - Cagliari 1.33 5.50 8.00   

### INGLATERRA
07:30 21 Nov V (0-2) -2.64 Newcastle - Chelsea 6.00 4.33 1.50   
10:00 21 Nov L 0.52 Aston Villa - Brighton 2.15 3.55 3.20   
12:30 21 Nov V -1.0 Tottenham - Manchester City 4.00 3.75 1.83   
15:00 21 Nov L 1.13 Manchester United - West Bromwich 1.37 5.00 8.00   



201118-(6000)
COPA COLOMBIA
16:55 1ª Mitad 0-0 V -0.51 Deportes Quindio - Deportivo Cali 3.45 2.90 2.25   
15:30 18 Nov V -1.12 Envigado FC - CD Junior 4.20 3.10 2.00   
17:45 18 Nov V -0.33 Rionegro Aguilas - Atletico Nacional 3.20 3.30 2.25   
20:00 18 Nov L 0.55 Independiente Santa Fe - America de Cali 2.15 3.00 3.75   
15:00 19 Nov E 0.05 Deportivo Pereira - Independiente Medellin 2.75 3.10 2.70   
19:40 19 Nov V -0.31 Alianza Petrolera - Millonarios 3.20 3.00 2.45   


20-11-17  DESPUES DE APUESTAS (WPLAY 67766)(CODERE 38519 unit)

201117-(7000 ) (4000)
ELIMINATORIAS
16:00 17 Nov V -0.08 Ecuador - Colombia 2.90 3.20 2.55   
16:00 17 Nov V -0.61 Venezuela - Chile 3.55 3.25 2.20   
18:00 17 Nov L 1.2 Paraguay - Bolivia 1.30 5.50 11.00   
18:00 17 Nov V -1.24 Uruguay - Brasil 4.333 3.30 1.95   
19:30 17 Nov V -2.0 Perú - Argentina 5.25 3.65 1.70   


20-11-15  DESPUES DE APUESTAS (WPLAY 77166)(CODERE 46639 unit)


20-11-14 (WPLAY 76166 8000 cu)(CODERE 59589 1000 cu x5 x6 x9 5000 unit)

COLOMBIA
13:30 14 Nov L 0.63 Patriotas Boyaca - Jaguares De Cordoba 2.05 3.20 3.90   
18:00 14 Nov V -0.31 Envigado FC - Independiente Medellin 3.20 3.10 2.375   
20:00 14 Nov V -0.12 Atletico Bucaramanga - Independiente Santa Fe 2.95 3.20 2.45   
18:00 15 Nov L 0.42 America de Cali - La Equidad 2.30 3.10 3.35   
18:00 15 Nov L 1.21 CD Junior - Boyaca Chico FC 1.25 6.00 11.00   
18:00 15 Nov L 0.75 Tolima - Once Caldas 1.909 3.10 4.50   
18:00 15 Nov V -0.3 Deportivo Pereira - Deportivo Pasto 3.20 2.80 2.60   
18:00 15 Nov L 1.22 Millonarios - Alianza Petrolera 1.30 5.00 11.00   
18:00 15 Nov L 0.62 Rionegro Aguilas - Deportivo Cali 2.05 3.20 3.85   

20-11-09
WPLAY FALTA 1
CODERE FALYAN 2
L 0.87 La Equidad Patriotas Boyaca 1.78 3.10 5.50   
L 0.7 Independiente Medellin Atletico Bucaramanga 1.96 3.20 4.30   
V -1.37 Deportivo Pereira Millonarios 4.50 3.20 1.909   
L 0.48 Independiente Santa Fe Tolima 2.25 2.85 3.75   
L 1.04 Deportivo Cali Envigado FC 1.53 4.00 6.50   
V -1.12 Jaguares De Cordoba CD Junior 4.20 2.95 2.05   
V -0.56 Alianza Petrolera Atletico Nacional 3.50 3.10 2.25   
XXE 0.06 Boyaca Chico FC Rionegro Aguilas 2.75 2.95 2.80   
XXL 0.81 Deportivo Pasto Once Caldas 1.85 3.10 5.25   
V -2.17 Cucuta Deportivo America de Cali 5.50 3.10 1.81   


PROCESO: 2020 11 05  

L 0.99 Independiente Santa Fe Independiente Medellin 1.61 3.55 6.25 (3-0) OK   
E 0.05 Envigado FC La Equidad 2.75 3.10 2.65   
L 0.94 Rionegro Aguilas Cucuta Deportivo 1.66 3.60 5.50   

L 0.41 Patriotas Boyaca Deportivo Pereira 2.35 2.75 3.75   
L 0.69 Atletico Bucaramanga Jaguares De Cordoba 1.96 3.40 4.00   
L 1.18 Tolima Boyaca Chico FC 1.364 4.50 9.50   
L 0.63 America de Cali Millionarios 2.05 3.20 3.95   
V -0.77 Once Caldas Deportivo Cali 3.75 3.20 2.10   
L 0.87 CD Junior Alianza Petrolera 1.80 2.95 6.00   
L 0.84 Atletico Nacional Deportivo Pasto 1.80 3.30 5.00   


L 1.0 Deportivo Cali Envigado FC 1.60 3.55 6.50   

L 0.41 Deportivo Pasto Once Caldas 2.30 3.30 3.10   



V -0.72 Boyaca Chico FC Rionegro Aguilas 3.70 3.10 2.10   
V -0.9 Alianza Petrolera Atletico Nacional 3.90 3.40 1.95   
L 0.93 La Equidad Patriotas Boyaca 1.666 3.60 5.50   
V -0.89 Deportivo Pereira Millionarios 3.90 3.25 2.00   
V -3.83 Cucuta Deportivo America de Cali 7.50 4.00 1.48   
L 0.5 Independiente Medellin Atletico Bucaramanga 2.20 3.10 3.50   
L 0.49 Independiente Santa Fe Tolima 2.20 3.30 3.30   
V -2.2 Jaguares De Cordoba CD Junior 5.50 3.60 1.666   

In [9]:
print ("COLOMBIA" ,"    ",'https://apuestas.wplay.co/es/t/19311/Colombia-Primera-A')
pronostico(pd.read_html('https://apuestas.wplay.co/es/t/19311/Colombia-Primera-A')[0])

print ("ESPAÑA","    ",'https://apuestas.wplay.co/es/t/19160/La-Liga')
pronostico(pd.read_html('https://apuestas.wplay.co/es/t/19160/La-Liga')[0])

print ("ITALIA","    ",'https://apuestas.wplay.co/es/t/19159/Serie-A')
pronostico(pd.read_html('https://apuestas.wplay.co/es/t/19159/Serie-A')[0])

print ("INGLATERRA","    ",'https://apuestas.wplay.co/es/t/19157/Inglaterra-Premier-League')
pronostico(pd.read_html('https://apuestas.wplay.co/es/t/19157/Inglaterra-Premier-League')[0])

print ("FRANCIA","    ",'https://apuestas.wplay.co/es/t/19327/Ligue-1')
pronostico(pd.read_html('https://apuestas.wplay.co/es/t/19327/Ligue-1')[0])



COLOMBIA      https://apuestas.wplay.co/es/t/19311/Colombia-Primera-A
39:02 1ª Mitad 2-1 L 0.98 Millonarios - Once Caldas 1.65 3.25 6.50      
16:00 02 Feb E 0.18 Envigado FC - Jaguares De Cordoba 2.625 2.70 3.25      
18:00 02 Feb V -1.15 Boyaca Chico FC - Deportivo Cali 4.25 2.85 2.11      
20:05 02 Feb V -0.22 Deportivo Pasto - Atletico Nacional 3.10 2.875 2.60      
14:00 03 Feb V -0.29 La Equidad - CD Junior 3.20 2.70 2.65      
16:00 03 Feb V -0.69 Deportivo Pereira - Tolima 3.65 3.25 2.09      
16:00 03 Feb L 1.1 Independiente Santa Fe - Patriotas Boyaca 1.45 4.25 7.50      
18:05 03 Feb E 0.22 Alianza Petrolera - Rionegro Aguilas 2.55 3.00 2.95      
20:10 03 Feb L 0.9 America de Cali - Atletico Bucaramanga 1.72 3.35 5.50      
20:00 04 Feb E 0.03 Independiente Medellin - Millonarios 2.80 2.75 2.90      
15:15 06 Feb L 0.74 Once Caldas - Envigado FC 1.95 2.90 5.00      
17:30 06 Feb L 1.08 CD Junior - Alianza Petrolera 1.50 3.75 7.50      
20:00 06 Feb L 1.3 Atletico Nacional 

In [6]:
def pronostico_doble(pd_fecha):
    for partido in range(len(pd_fecha)):
        
        local_list=pd_fecha[3][partido].split()
        empate_list=pd_fecha[5][partido].split()
        visitante_list=pd_fecha[7][partido].split()
        
        if local_list[0]=="★":
        
            cuota_l=local_list[-2]
            cuota_e=empate_list[-2]
            cuota_v=visitante_list[-2]

            equipo_local=equipo(local_list)
            equipo_visitante=equipo(visitante_list)
    
            loc=float(cuota_l)
            emp=float(cuota_e)
            vis=float(cuota_v)
    
            pronostico=-0.805*loc-0.0517*emp+0.0125*vis+2.3923
    
            umbral=0.3
    
            if pronostico>umbral:
                gana="L"
            elif pronostico<0:
                gana="V"
            else:
                gana="E"
    
            print (pd_fecha[1][partido],gana,round(pronostico,2),equipo_local,"-",equipo_visitante,cuota_l,cuota_e,cuota_v,"     ")
